sources/Application/Instruments/MidiInstrument.h

```diff
@@ -80,6 +80,15 @@ class MidiInstrument : public I_Instrument {
   char velocity_ = 127;
   TableSaveState tableState_;
   bool first_[SONG_CHANNEL_COUNT];
+  bool pitchBend_;
+  int pitchBendTarget_;
```

Comment:
please use specific int sizes, eg. I think this one you cast to 8bit in the cpp file